core/src/organization/database/member.repository.ts

```diff
@@ -0,0 +1,26 @@
+import {Inject, Injectable} from "@nestjs/common";
+import {DataSource} from "typeorm";
+
+import {ExtendedRepository, ProfiledRepository} from "../../types/extended-repositories";
+import {Member} from "./member.entity";
+import type {MemberProfile} from "./member-profile.entity";
+import {MemberProfileRepository} from "./member-profile.repository";
+
+@Injectable()
+export class MemberRepository extends ExtendedRepository<Member> {
+    constructor(readonly dataSource: DataSource) {
+        super(Member, dataSource);
+    }
+}
+
+@Injectable()
+export class MemberProfiledRepository extends ProfiledRepository<Member, MemberProfile> {
+    readonly profileInverseRelationshipName: "member";
+
+    constructor(
+        @Inject(MemberRepository) readonly primaryRepository: MemberRepository,
+        @Inject(MemberProfileRepository) readonly profileRepository: MemberProfileRepository,
+    ) {
+        super();
+    }
+}
```

Comment:
I don't like having two separate repositories for one concept (Member). Can we combine these somehow? Or maybe `ProfiledRepository` should extend `ExtendedRepository`?

Reply:
It can't extend because it uses two models. There also is a member profile repository. I may have just forgot to move it.